imageresizerapp/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,7 @@
+﻿Install-ChocolateyPackage `
+    -PackageName 'imageresizerapp' `
+    -FileType 'EXE' `
+    -SilentArgs '/quiet /norestart' `
+    -Url 'https://github.com/bricelam/ImageResizer/releases/download/v3.0/ImageResizerSetup-3.0.exe' `
```

Comment:
In this case, then, providing the `64` params is cosmetic. I suspect anyone using `-forcex86` is doing it wrong :stuck_out_tongue: